upload-api/functions/validate-email.jsx

```diff
@@ -134,11 +144,21 @@ export async function validateEmailPost (request) {
 
   const { email, audience, ucan } = authorizeResult.ok
 
+  const planCheckResult = await context.customerStore.get({ customer: DidMailto.fromEmail(email) })
+  let stripePricingTableId
+  let stripePublishableKey
+  if (!planCheckResult.ok?.product){
```

Comment:
is the idea here to only show the pricing table in the case where that account hasn't already chosen one?

Reply:
Yep exactly. This could be cleaner but I think we will move this all to
console sooner rather than later